py/src/ai/h2o/sparkling/Initializer.py

```diff
@@ -43,7 +44,11 @@ class Initializer(object):
     @staticmethod
     def load_sparkling_jar():
         sc = SparkContext._active_spark_context
-        if Initializer.__sparklingWaterJarLoaded is not True and sc is not None:
+        if Initializer.__sparklingWaterJarLoaded is False and sc is None:
```

Comment:
nit: Maybe split the conditions into two `if` statements:
```
if Initializer.__sparklingWaterJarLoaded is False:
  if sc is None:
    ...
  else:
    ...
```

Reply:
👍